service/controllers.py

```diff
@@ -693,9 +690,16 @@ def post(self):
         else:
             context = {'error': response,
                        'username': session.get('username'),
-                       'mfa_token_name': mfa_token_name}
+                       'mfa_token_name': self.create_token()}
```

Comment:
**What?** Do not reuse same token field name. Create a new one, so browser does not try to autofill with past value.